specification/communication/data-plane/Identity/stable/2022-10-01/CommunicationIdentity.json

```diff
@@ -47,7 +61,7 @@
           }
         },
         "x-ms-examples": {
-          "Create an Identity and optionally an access token with custom expiration time within the [60-1440] minutes range. If expiration time is not specified, the default value of 1440 minutes (24 hours) will be used.": {
+          "Create an Identity and optionally an access token with optional idempotent processing using Repeatability-Request-ID and Repeatability-First-Sent headers. If the headers are not provided, the request will be processed in a non-idempotent way.": {
```

Comment:
```suggestion
          "Create an Identity and optionally an access token with optional idempotent processing using Repeatability-Request-ID and Repeatability-First-Sent headers. If the headers are not provided, the request will be processed in a non-idempotent way which can lead to more users being created than intended on a retry.": {
```

Reply:
Formulation added.

Reply:
Thanks one minor thing that I missed when making the suggestion: Let's say "more identities" instead of "more users"